gnucash/import-export/aqb/gschemas/org.gnucash.dialogs.flicker.gschema.xml.in

```diff
@@ -0,0 +1,24 @@
+<schemalist gettext-domain="@PROJECT_NAME@">
+
+  <schema id="org.gnucash.dialogs.flicker" path="/org/gnucash/dialogs/flicker/">
+    <key name="window-size" type="(ii)">
+      <default>(-1,-1)</default>
+      <summary>Last window size</summary>
+      <description>This setting describes the size of the window when it was last closed.
+        The numbers are the width and height of the window. </description>
+    </key>
+    <key name="barwidth" type="u">
+      <default>44</default>
+      <summary>Last value of the barwidth</summary>
+      <description>This value is the setting for the "Bar Width" field.
+        The user can adjust the distance between the flicker bars to the sensors of his TAN generator.</description>
+    </key>
+    <key name="delay" type="u">
+      <default>50</default>
+      <summary>Sets the delaytime of the flickering grafic</summary>
+      <description>This setting changes the flashing speed of the flicker display.
+        The display changes faster with smaller values.</description>
```

Comment:
Ideally the description reuses the text from the corresponding tooltip in the glade file or in this case tooltip_text from gnc-flicker-gui.c.
BTW If you do no concatenation of strings, it is easier to set them already in the glade file, see above.

Reply:
@fellen: the schema file is old. After a tip from Bob-IT it is sufficient here, the window position and size to save. The user values are saved and loaded with a different mechanism.
Can I edit the file directly in commit?